src/main/java/gift/vo/Product.java

```diff
@@ -0,0 +1,58 @@
+package gift.vo;
+
+public class Product {
+
+    private Long id;
+    private String name;
+    private int price;
+    private String imageUrl;
+
+    public Product() { }
+
+    public Product(Long id, String name, int price, String imageUrl) {
+        this.id = id;
+        this.name = name;
+        this.price = price;
+        this.imageUrl = imageUrl;
+    }
+
+    public Product(String name, int price, String imageUrl) {
+        this.name = name;
+        this.price = price;
+        this.imageUrl = imageUrl;
+    }
```

Comment:
생성자 체이닝을 활용해보아도 좋겠네요!

```suggestion
    public Product(String name, int price, String imageUrl) {
        this(null, name, price, imageUrl);
    }

    public Product(Long id, String name, int price, String imageUrl) {
        this.id = id;
        this.name = name;
        this.price = price;
        this.imageUrl = imageUrl;
    }
```